### ISSUE TITLE
Pagination buttons have no label element

### ISSUE THREAD

**Issue by amortiz** [carbon-design-system/carbon#7233]
go here: https://react.carbondesignsystem.com/?path=/story/pagination--multiple-pagination-components

Pagination "previous" and "next" buttons have no label element. This seems like regression as we didn't have that before.

![image](https://user-images.githubusercontent.com/10033254/98296527-ae8dd780-1f78-11eb-8575-0f0203f343ae.png)

**Comment by emyarod** [comment]
looks like this is a regression related to #7021 and #7029
